tests/ssl-opt.sh

```diff
@@ -882,6 +900,11 @@ print_name() {
     fi
 
     LINE="$LINE$1"
+    if [ "$LIST_TESTS" -gt 0 ]; then
```

Comment:
`--list-test-cases` doesn't actually work with `--number`, so you might as well not call `print_name` at all in list mode.

(Or fix it so that `ssl-opt.sh --list-test-cases --number` works, but I don't think that's important.)